src/components/Features/Transfer/Transfer.js

```diff
@@ -110,6 +116,15 @@ export const Transfer = () => {
     }
   };
 
+  const BridgeIsFullError = () => {
```

Comment:
This kind of title is a components convention.
Change to `renderBridgeIsFullError`

Reply:
But `BridgeIsFullError` is a component

Reply:
So place it outside of the `Transfer` component.

Reply:
done